ci_framework/roles/env_op_images/tasks/main.yml

```diff
@@ -0,0 +1,104 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Ensure directory is present
+  ansible.builtin.file:
+    path: "{{ cifmw_env_op_images_dir }}/{{ item }}"
+    state: directory
+  loop:
+    - artifacts
+    - logs
+
+- name: Check for oc command
```

Comment:
Not needed as https://github.com/openstack-k8s-operators/ci-framework/pull/772/files#diff-ea5415b13a2e225e67af327bdb8491a82cda390948f8b9f6c36eacad6ecf38edR34 will fail if oc is not present and the entire block will be skipped. I'd add a check in the block to test that `openstack_installed.rc == 0`